local/compose.go

```diff
@@ -366,7 +366,11 @@ func getContainerCreateOptions(p *types.Project, s types.ServiceConfig, number i
 	labels := map[string]string{
 		projectLabel:         p.Name,
 		serviceLabel:         s.Name,
+		versionLabel:         ComposeVersion,
+		oneoffLabel:          "False",
```

Comment:
Should we add a "TODO" here? I feel like this shouldn't be a constant here

Reply:
Yes, that will be true for "docker compose run" containers I think... I'll add that

Reply:
that's also needed so that contianer created by `docker compose up` can be excluded by docker-compose one_off filters, so still relevant here